src/app/routes/_question.router.js

```diff
@@ -70,6 +71,60 @@ export default (app, router, io, admin) => {
   });
 
   /**
+   * @api {post} /api/question Get suggestions for question text from other questions and youtube video search
+   * @apiName SuggestQuestions
+   * @apiGroup Question
+   *
+   * @apiParam {String} text Question body
+   *
+   * @apiSuccess {Object[]} suggestions            List of suggestions
+   * @apiSuccess {Array}    suggestions.questions  List of related questions (already answered, with video link)
+   * @apiSuccess {Array}    suggestions.youtube    List of related Youtube videos
+   */
+  router.post('/api/question/suggestions', async (req, res) => {
+    // requires index: db.getCollection('questions').createIndex( { text: "text" } )
+    try {
+      const keywords = req.body.text.match(/\w+/g);
+      const youtubeData = await youtubeHelper.findMatchingVideos(keywords.join('|'), 8);
+      const questions = await Question
+        .find({
+          videoUrl: { $exists: true },
```

Comment:
`Question`'s `videoUrl` is unused. You need to query for `videoUrl` of a question's referenced broadcast.